NuGet.config

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
NuGet is the .NET package manager (like NPM for node). This file defines which servers should be included for the package restore project.

As long as the file only contains this one default endpoint, it would not be necessary. But I like having it in the repo because it also has an explicit "clear" statement. This means, the repo still takes the default server, even if someone has a machine-level override.